src/agent/main.cpp

```diff
@@ -154,7 +154,7 @@ static int realmain(int argc, char *argv[])
         switch (opt)
         {
         case OTBR_OPT_BACKBONE_INTERFACE_NAME:
-            backboneInterfaceName = optarg;
+            backboneInterfaceNames.push_back(optarg);
```

Comment:
It's alive across the whole program lifetime.

Reply:
I had the same question (since I saw we used`vector<const char *>`). I guess `optarg` is always coming from `argv` array which remains valid in `main()`